docs/xamarin-forms/app-fundamentals/localization/text.md

```diff
@@ -634,8 +634,8 @@ namespace UsingResxLocalization.Android
 Once this code has been added to the Android application project, it will be able
 to automatically display translated strings.
 
-> [!NOTE]
```

Comment:
This was in alert format, but was a warning inside

Reply:
oops!